mlir/lib/Target/LLVMIR/Dialect/OpenMP/OpenMPToLLVMIRTranslation.cpp

```diff
@@ -289,7 +281,16 @@ static LogicalResult checkImplementationStatus(Operation &op) {
         checkBare(op, result);
         checkDevice(op, result);
         checkHasDeviceAddr(op, result);
-        checkHostEval(op, result);
+
+        // Host evaluated clauses are supported, except for target SPMD loop
```

Comment:
Nit: Would it make sense to keep this segment inside of the checkHostEval function just to keep things "tidy"? I'll leave it up to you, not overly fussed personally!